### ISSUE TITLE
srs sampling w/ replacement

### ISSUE THREAD

**Issue by paul-vdb** [tmcd82070/SDraw#26]
In srs,point samples are drawn from ind <- ceiling(runif(n, 0, N)). There is not an option to choose without vs with replacement but inherently this implies for sampling with replacement. Sampling points in WA.cities n = 50 results in a fair chance of repeated samples being added. Exported data treats these as "new" samples and is confusing in the output until digging deeper. If intentional please make it obvious and if not please add a catch for replacement to achieve n actual samples. Thanks!

**Comment by tmcd82070** [comment]
True. A good point. Sampling is with replacement when applied to point resources, which are finite.  Best thing to do will be to add an option to select with or without replacement.  And, need to make this explicit in the xxx.point sampling method documentation.
